src/Providers/DataGrid/Wijmo/Features/ContextMenu.ts

```diff
@@ -60,6 +60,10 @@ namespace Providers.DataGrid.Wijmo.Feature {
 
 			//Sort menu by order - Usefull when the developer inserts a IF statement hiding/showing elements
 			this._sortMenuItems(this._rootMenuItems);
+			//If the menu is opening, let's refresh the itemsSource
+			if (this._isOpening) {
+				this._provider.itemsSource.refresh();
+			}
```

Comment:
Why do we need to refresh the itemsSource here?

Reply:
So that if an element is added by the developer when the ContextMenu is opening, the ContextMenu reflects that change.